src/web/core/src/main/java/org/geoserver/web/data/layer/AttributeEditPage.html

```diff
@@ -19,10 +19,11 @@
     </div>
     <div wicket:id="sizeContainer" class="mb-3">
       <label for="size"><wicket:message key="size">size</wicket:message></label>
-      <input id="size" class="text" type="text" wicket:id="size"/></div>
+      <input id="size" class="text" type="text" wicket:id="size"/>
+    </div>
     <div wicket:id="crsContainer" class="mb-3">
       <label for="crs"><wicket:message key="crs">cRS</wicket:message></label>
-      <span wicket:id="crs"></span></div>
+      <span wicket:id="crs"></span>
```

Comment:
When creating new feature type from ui (when publishing new layer) it crashed due to 1 more closing tag then needed